versioned/storage/jdbc-tests/src/main/java/org/projectnessie/versioned/storage/jdbctests/MySQLBackendTestFactory.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.storage.jdbctests;
+
+import jakarta.annotation.Nonnull;
+import org.projectnessie.versioned.storage.jdbc.JdbcBackendFactory;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MySQLContainer;
+
+public class MySQLBackendTestFactory extends ContainerBackendTestFactory {
+
+  @Override
+  public String getName() {
+    return JdbcBackendFactory.NAME + "-MySQL";
+  }
+
+  @Nonnull
+  @Override
+  protected JdbcDatabaseContainer<?> createContainer() {
+    String dockerImage = dockerImage("mysql");
+    return new MariaDBDriverMySQLContainer(dockerImage);
+  }
+
+  /** This has been done so that mariaDB Driver can be used for mysql Testcontainer */
```

Comment:
You have a checkstyle violation here:

```
MySQLBackendTestFactory.java:37: First sentence of Javadoc is missing an ending period. [SummaryJavadoc]
```

In fact I think you can remove this javadoc completely as the class code itself is quite self-explanatory.